[EDU-4234] add revalidated status to edge cache doc (#868)
aziontech/docs: src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-caching.mdx

Azion provides a layer to cache content in the edge network, which stands between origin servers and the end user. When content is successfully cached at the edge, it receives a [cache status HIT](#status-definitions) and it can be directly delivered to your users from the edge node nearest to them, reducing the frequency of access to origin servers.

> When the requested content is not available in the cache, Azion's Edge Cache module employs strategies to minimize the impact on the origin server and optimize content delivery. One of these strategies includes maintaining keep-alive connections with the origin server whenever possible, to reduce the overhead associated with frequent TCP/IP handshakes. This is particularly effective during cache updates or in situations of a [cache MISS](#status-definitions).

Reviewer: ```suggestion
When the requested content isn't available in the cache, Azion's Edge Cache module employs strategies to minimize the impact on the origin server and optimize content delivery. One of these strategies includes maintaining keep-alive connections with the origin server whenever possible, to reduce the overhead associated with frequent TCP/IP handshakes. This is particularly effective during cache updates or in situations of a [cache MISS](#status-definitions).
```

> When the requested content is not available in the cache, Azion's Edge Cache module employs strategies to minimize the impact on the origin server and optimize content delivery. One of these strategies includes maintaining keep-alive connections with the origin server whenever possible, to reduce the overhead associated with frequent TCP/IP handshakes. This is particularly effective during cache updates or in situations of a [cache MISS](#status-definitions).

Reviewer 2: ```suggestion
When the requested content isn't available in the cache, the Azion Edge Cache module employs strategies to minimize the impact on the origin server and optimize content delivery. One of these strategies includes maintaining keep-alive connections with the origin server whenever possible, to reduce the overhead associated with frequent TCP/IP handshakes. This is particularly effective during cache updates or in situations of a [cache MISS](#status-definitions).
```

> When the requested content isn't available in the cache, the Azion Edge Cache module employs strategies to minimize the impact on the origin server and optimize content delivery. One of these strategies includes maintaining keep-alive connections with the origin server whenever possible, to reduce the overhead associated with frequent TCP/IP handshakes. This is particularly effective during cache updates or in situations of a [cache MISS](#status-definitions).

Author: ```suggestion
When the requested content isn't available in the cache, the Azion Edge Cache add-on employs strategies to minimize the impact on the origin server and optimize content delivery. One of these strategies includes maintaining keep-alive connections with the origin server whenever possible, to reduce the overhead associated with frequent TCP/IP handshakes. This is particularly effective during cache updates or in situations of a [cache MISS](#status-definitions).
```

Já é pra mudar?

| **EXPIRED** | The cached content has expired the TTL set at the edge. When the origin responds, the cached content is updated for future requests |
| **STALE** | When choosing to serve [stale cache](#stale-cache), if the origin fails to respond and the edge cache has expired, a stale version of the content is served |
| **UPDATING** | The cached content has expired and a stale cache is being served, but the content is being updated in the origin |

> | **REVALIDATED** | The cached content has expired or has been previously served as stale, but hasn't been modified since the time it was recorded. If the cache is still valid, the edge returns this status and won't request the full content from the origin |

Author: ```suggestion
| **REVALIDATED** | The cached content is checked against the origin using conditional headers. If it's still up-to-date, the resource is not retransmitted. |
```
